.github/workflows/terraform-dev.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       # Build Docker image.
       - name: Docker Image
         id: build
-        run: docker build .. -t near/mpc-recovery
+        run: docker build ../.. -t near/mpc-recovery
```

Comment:
@kmaus-near Why do we need to build this image at all? Aren't we using Artifact Registry one in the next step?